source/fem-dem/fluid_dynamics_vans_matrix_free_operators.cc

```diff
@@ -170,17 +267,26 @@ VANSOperator<dim, number>::compute_particle_fluid_force(
 }
 
 /**
+ * Calculates the Jacobian of VANS equations. Here, we note  f_pf
+ * the forces from the particles to the fluid.
  * The expressions calculated in this cell integral are:
- * (q,ε∇·δu) + (q,δu·∇ε)  (Continuity equation)
- * \+ (v, ε ∂t δu) + (v, ε (u·∇)δu) + (v, ε (δu·∇)u)
- * \- (∇·v, ε δp) - (v,p∇ε)
- * \+ ε*ν(∇v,∇δu)  +  ν(v,∇ε·∇δu)   (VANS equations),
- * plus three additional terms in the case of SUPG-PSPG stabilization:
- * \+ (ε ∂t δu + ε(u·∇)δu +  ε(δu·∇)u +  ε∇δp -  εν∆δu)τ·∇q (PSPG
+ * (Continuity equation)
+ * (q,ε∇·δu) + (q,δu·∇ε) +
+ * (VANS equations)
+ * \+ (v, ε ∂t δu)  -> Time derivative
+ * \+ (v, ε (u·∇)δu) + (v, ε (δu·∇)u) -> Advection
+ * \- (∇·v, ε δp) - (v,p∇ε) -> Pressure
```

Comment:
```suggestion
 * \- (∇·v, ε δp) - (v,p∇ε) -> Pressure
```
Should it be delta p in the second term?